apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/account/dao/AccountGuaranteedBalance.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright © 2018-2019 Apollo Foundation
+ */
+
+package com.apollocurrency.aplwallet.apl.core.account.dao;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import com.apollocurrency.aplwallet.apl.core.db.DbKey;
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Entity class
+ */
+@Setter @Getter
+public final class AccountGuaranteedBalance {
```

Comment:
I guess It might be inherited DerivedEntity class

Reply:
I tried to do that, but looks like that is bad idea in that case. No sense in that case.